common/lib/xmodule/xmodule/js/src/video/10_main.js

```diff
@@ -139,6 +140,10 @@
                     initialize(state, element);
                 }
 
+                if (autoadvance_enabled) {
+                    mainVideoModules.push(VideoAutoAdvanceControl);
+                }
```

Comment:
Added equivalent comment for the most recent diff: https://github.com/edx/edx-platform/pull/15803#discussion_r134070345